### DISCUSSION TITLE
Aurora AOne In-Line Dimmer Module AU-A1ZB320

### DISCUSSION THREAD

**Discussion by Daverover66** [Koenkk/zigbee2mqtt#18590] [Devices]
Has anyone successfully added this to their zigbee2mqtt in Home Assistant? I tried, and while it pairs okay, it's unsupported. I tried the "support new devices" guide and had a go at using the config for an Aurora device that is supported, but all I got was an error "zigbee2mqtt cannot read properties of undefined (reading 'includes')".  I have very limited knowledge of mqtt but perhaps naively presumed that as the Aurora AU-A1ZBRC remote control, which can control this dimmer module, is supported, that the module itself would be. Now I find myself unable to get this dimmer module to connect to Home Assistant.

It didn't help that once the dimmer module is paired with zigbee2mqtt I have no way of triggering actions to produce debug messages in the logs, as the device is unsupported. Most frustrating.  

Is there any way requesting support for the device be added?

**Comment by x10** [comment]
I found another post on this here, it seems it is only exposing one action, so its not able to do all of (on/off presses and dim up/dim down) for the [Lutron Aurora](https://www.zigbee2mqtt.io/devices/Z3-1BRL.html) as well.

 https://community.home-assistant.io/t/making-use-of-lutron-aurora-using-mqtt-payload-to-control-light-group/605543
